modules/instrument_list/templates/menu_instrument_list.tpl

```diff
@@ -88,7 +110,7 @@
         {$display.PSC}
       </td>
       <td>
-        {$display.CohortTitle}
+        {$display.SubprojectTitle}
```

Comment:
This looks like a step backwards/a bug?

I thought all references to "Subproject" were replaced by "Cohort"

Reply:
Oh sorry, I am not sure how this change snuck through